index.js

```diff
@@ -144,20 +135,6 @@ SmartBanner.prototype = {
 			path: '/',
 			expires: +new Date() + this.options.daysReminder * 1000 * 60 * 60 * 24
 		});
-	},
```

Comment:
@jbbuckner I'd be tempted to leave this stuff in. we don't want to take out functionality from the existing plugin, but just build upon it.